src/elm/clinical.ts

```diff
@@ -158,11 +168,13 @@ export class Code extends Expression {
   version: string;
   display?: string;
 
-  constructor(json: any) {
+  constructor(json: ELM.Code) {
     super(json);
     this.code = json.code;
-    this.systemName = json.system.name;
-    this.version = json.version;
+    this.systemName = json.system?.name || '';
+    // @ts-ignore
+    // TODO: Spec says version does not exist on code?
```

Comment:
you're right.... @cmoesel ??

Reply:
Notably though, this breaks [this test](https://github.com/cqframework/cql-execution/blob/3b667a6f0cca2fd6d1b4772f9457f70325e866d0/test/elm/clinical/clinical-test.ts#L168), and I think snowball effects into [this one](https://github.com/cqframework/cql-execution/blob/3b667a6f0cca2fd6d1b4772f9457f70325e866d0/test/elm/clinical/clinical-test.ts#L186) too

Reply:
Oops, and [this one](https://github.com/cqframework/cql-execution/blob/3b667a6f0cca2fd6d1b4772f9457f70325e866d0/test/elm/clinical/clinical-test.ts#L209) and maybe [this one](https://github.com/cqframework/cql-execution/blob/3b667a6f0cca2fd6d1b4772f9457f70325e866d0/test/elm/parameters/parameters-test.ts#L188)

They all have to deal with a default assignment to version 🤔